socorro-cmd

```diff
@@ -91,16 +91,14 @@ COMMANDS = [
     ),
     Group(
         'Local development environment maintenance', {
-            'create_recent_indices': import_path(
-                'socorro.external.es.create_recent_indices_app.CreateRecentESIndicesApp'
-            ),
             'clear_indices': import_path(
                 'socorro.external.es.clear_indices_app.ClearESIndicesApp'
             ),
             'upload_telemetry_schema': import_path(
                 'socorro.external.boto.upload_telemetry_schema.UploadTelemetrySchema'
             ),
             'db': import_path('socorro.scripts.db.main'),
+            'es': import_path('socorro.scripts.es.main'),
```

Comment:
This script is like db and pubsub--it manages create and delete of the resource. We can add other sub sub commands, too. For now, the es one only does create.